src/docs/cookbook-events/like-feature/index.md

```diff
@@ -0,0 +1,527 @@
+---
+title: How to implement a like feature using events
+slug: like-feature
+updated: 2021-01-20
+category: cookbook-events
+ingress:
+  Following this guide, you'll be able to build a feature that allows users to like listings using events and user extended data
+published: true
+---
+
+Events represent changes in marketplace data resources. In this guide, we use events to listen to changes in user extended data. We react to these events by updating a 'likes' value in listing extended data. In addition, we display the number of likes on the listing page and allow users to interact by clicking on a 'like' -button. 
+
+You might be wondering why we don't we just store the 'likes' directly to listing extended data. There are two reasons for that: saving likes in user extended data tells us what listings a user has liked. In addition, we avoid a race condition caused by two users liking a listing simultaneously.
+
+A downside to using events is that the like count may take a moment to update. In this guide, we temporarily increment the like value in the UI to hide the delay. 
+
+You can download the files used in this guide from our example repository here, or follow along using the embedded code snippets.
+
+## Create a UI component the user can interact with
+We start by creating an icon that users can interact with to like or dislike a listing. Next to the icon, we'll display the number of likes. For the UI component, we'll create a new subcomponent `SectionLikes.js` in the ListingPage directory. 
+
+###### Step 1: Create a new file
+
+```shell
+└── src
+    └── containers
+        └── ListingPage
+            ├── SectionLikes.js
+            ├── ListingPage.js
+            └── ListingPage.module.css
+```
+
+We'll create a new component that renders an svg and the amount of likes:
```

Comment:
This might be a matter of preference, but for me it's easier to read a guide like this if there's some explanation between the heading and the code example, so there's some context as to what I should be paying attention to.

Reply:
Added some explanations below the H5 headings!